src/models/analytics/analytics-tables.ts

```diff
@@ -429,6 +429,7 @@ const analyticsTables: Record<TableKeys, AnalyticsTable> = {
         level: 'Condition',
         deviceFields: 'Device form fields',
         deviceField: 'Device form field',
+        field: 'Form field',
```

Comment:
I was creating an analytics query with pin and then a form field that gave a error in the console. Since it has not yet caused it's own sentry ticket I thought add it in here so it moves to production quickly.